django_project/certification/models/certificate.py

```diff
@@ -0,0 +1,54 @@
+# coding=utf-8
+"""
+Certificate model definitions for certification apps
+"""
+
+import random
+import string
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.text import slugify
+from core.settings.contrib import STOP_WORDS
+
+
+class Certificate(models.Model):
+    """Certificate model."""
+
+    certificateID = models.CharField(
+        help_text="Id certificate.",
+        max_length=200,
+        blank=False,
+        null=False,
+    )
+
+    slug = models.SlugField()
+    objects = models.Manager()
+
+    class Meta:
+        """ Meta class for Certificate."""
+
+        ordering = ['certificateID']
+
+    def __unicode__(self):
+        return u'%s' % self.id_id
+
+    def save(self, *args, **kwargs):
+
+        if not self.pk:
+            certificateID = self.slug_generator()
+            words = certificateID.split()
+            filtered_words = [t for t in words if t.lower() not in STOP_WORDS]
```

Comment:
In general don't use variable name with 1 character, except it's obvious (like i for index).